src/pybind/mgr/dashboard/controllers/health.py

```diff
@@ -105,16 +105,25 @@ def df(self):
     def fs_map(self):
         fs_map = mgr.get('fs_map')
         if self._minimal:
+<<<<<<< HEAD
             fs_map = self._partial_dict(fs_map, ['filesystems', 'standbys'])
             fs_map['standbys'] = [{}] * len(fs_map['standbys'])
             fs_map['filesystems'] = [self._partial_dict(item, ['mdsmap']) for
+=======
+            fs_map = partial_dict(fs_map, ['filesystems', 'standbys'])
+            fs_map['filesystems'] = [partial_dict(item, ['mdsmap']) for
+>>>>>>> cdfeb1d196... mgr/dashboard: add popover list of managers in landing page
                                      item in fs_map['filesystems']]
             for fs in fs_map['filesystems']:
                 mdsmap_info = fs['mdsmap']['info']
                 min_mdsmap_info = dict()
                 for k, v in mdsmap_info.items():
+<<<<<<< HEAD
                     min_mdsmap_info[k] = self._partial_dict(v, ['state'])
                 fs['mdsmap'] = dict(info=min_mdsmap_info)
+=======
+                    min_mdsmap_info[k] = partial_dict(v, ['state'])
+>>>>>>> cdfeb1d196... mgr/dashboard: add popover list of managers in landing page
```

Comment:
Hey @avanthakkar, looks like you haven't properly resolved a git conflict. If you don't know what this mean or how to fix it, please let me know and I'll help.

Reply:
You talking about the whitespace added? I think the changes are reverted which I made. You can check with https://github.com/ceph/ceph/pull/33529 , there are few lines which I removed but they are not reflected here. So how can I make the changes and reflect here?

Reply:
Nope, I mean `<<<<<<< HEAD` or `=======`. These are referred as [merge conflict markers](https://wincent.com/wiki/Understanding_Git_conflict_markers).

Reply:
Yeah I got that. So the changes I made in upstream for this part of code is causing the merge conflict , right?

Reply:
I think the changes I made in upstream branch are not same as that in the backported branch